monitoring/base/rhods-rules.yaml

```diff
@@ -16,5 +16,9 @@ spec:
       expr: count(kube_pod_container_status_ready{namespace="rhods-notebooks",pod=~"jupyterhub-nb.*"}==1)
     - record: cluster:usage:consumption:rhods:active_users
       expr: count(count_over_time(kube_pod_container_status_ready{namespace="rhods-notebooks",pod=~"jupyterhub-nb.*"}[1h]))
+    - record: cluster:usage:consumption:rhods:cpu_requests_runtime
+        expr: sum_over_time(sum(kube_pod_container_resource_requests{namespace='rhods-notebooks', resource='cpu'})[30d:1h])
```

Comment:
Sorry I didn't clarify this before - this expression (and the one below) should not have the sum_over_time part, we'll do that on the backend for the billing calculation.

Reply:
Done, updated to have just the sum.